pages/productList/productlist.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/b84d0d614d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/doc_style/navbar.css" />
+    <link rel="stylesheet" href="/styles.css" />
+    <title>productlist</title>
+  </head>
+  <body>
+    <div class="ecom-container">
+      <!--Navigation-->
+      <nav class="navbar flex-row">
+        <a href="">
+          <h1 class="nav-heading">
+            one8<small class="nav-heading-small">Store</small>
+          </h1>
+        </a>
+        <div class="nav-search flex-row">
+          <i class="fas fa-search search-icon"></i>
+          <input type="text" class="nav-input" placeholder="Type to search" />
+        </div>
+
+        <div>
+          <a href=""><i class="fas fa-heart icon-md nav-icon"></i></a>
+          <a href=""><i class="fas fa-shopping-cart icon-md nav-icon"></i></a>
+        </div>
+      </nav>
+
+      <main class="main flex-row">
+        <!--Sidebar-->
+        <aside class="filters flex-col">
+          <div class="sub-section flex-row">
+            <h3 class="heading">Filters</h3>
+            <p>Clear All</p>
+          </div>
+          <hr class="horizontal-line" />
+          <h3 class="heading">Price</h3>
+          <input type="range" class="range" />
+          <hr class="horizontal-line" />
+          <h3 class="heading">Category</h3>
+          <ul class="list">
+            <li>
+              <input type="checkbox" />
+              <span>Sneakers</span>
+            </li>
+            <li>
+              <input type="checkbox" />
+              <span>T-shirt</span>
+            </li>
+            <li>
+              <input type="checkbox" />
+              <span>Perfumes</span>
+            </li>
+          </ul>
+          <hr class="horizontal-line" />
+          <h3 class="heading">Rating</h3>
+          <ul class="list">
+            <li>
+              <input type="radio" name="rating" />
+              <span>4 stars & above</span>
+            </li>
+            <li>
+              <input type="radio" name="rating" />
+              <span>3 stars & above</span>
+            </li>
+            <li>
+              <input type="radio" name="rating" />
+              <span>2 stars & above</span>
+            </li>
+            <li>
+              <input type="radio" name="rating" />
+              <span>1 stars & above</span>
+            </li>
+          </ul>
+          <hr class="horizontal-line" />
+          <h3 class="heading">Sort By</h3>
+          <ul class="list">
+            <li>
+              <input type="radio" />
+              <span>Price-Low to High</span>
+            </li>
+            <li>
+              <input type="radio" />
+              <span>Price-High to Low</span>
```

Comment:
Add name attribute. Due to missing of this attribute, you can select both filter simultaneously which does not make sense.

Reply:
make these changes before merging the pr